#!/usr/bin/python3
>>> add_integer = __import__('0-add_integer').add_integer
>>> add_integer(1, 2)
3
>>> add_integer(100, -2)
98
>>> add_integer(2)
100
>>> add_integer(100.3, -2)
98

>>> add_integer("School", 1)
Traceback (most recent call last):
    ...
TypeError: a must be an integer

>>> add_integer(4, "School")
Traceback (most recent call last):
    ...
TypeError: b must be an integer

>>> add_integer(None)
Traceback (most recent call last):
    ...
TypeError: a must be an integer

>>> add_integer(5, None)
Traceback (most recent call last):
    ...
TypeError: b must be an integer

>>> add_integer([1, 2, 3], 4)
Traceback (most recent call last):
    ...
TypeError: a must be an integer

>>> add_integer(1, {"num": 5})
Traceback (most recent call last):
    ...
TypeError: b must be an integer

>>> add_integer((3, 4), 1)
Traceback (most recent call last):
    ...
TypeError: a must be an integer

>>> add_integer(1, (3, 4))
Traceback (most recent call last):
    ...
TypeError: b must be an integer

>>> add_integer(1, 2)
3
>>> add_integer(100, -2)
98
>>> add_integer(2)
100
>>> add_integer(100.3, -2)
98

>>> add_integer("School", 1)
Traceback (most recent call last):
    ...
TypeError: a must be an integer

>>> add_integer(4, "School")
Traceback (most recent call last):
    ...
TypeError: b must be an integer

>>> add_integer(None)
Traceback (most recent call last):
    ...
TypeError: a must be an integer

>>> add_integer(5, None)
Traceback (most recent call last):
    ...
TypeError: b must be an integer

>>> add_integer([1, 2, 3], 4)
Traceback (most recent call last):
    ...
TypeError: a must be an integer

>>> add_integer(1, {"num": 5})
Traceback (most recent call last):
    ...
TypeError: b must be an integer

>>> add_integer((3, 4), 1)
Traceback (most recent call last):
    ...
TypeError: a must be an integer

>>> add_integer(1, (3, 4))
Traceback (most recent call last):
    ...
TypeError: b must be an integer

>>> add_integer(100, -2)
98

>>> add_integer(1, 2)
3

>>> add_integer(2)
100

>>> add_integer(100.3, -2)
98

>>> add_integer("School", 1)
Traceback (most recent call last):
    ...
TypeError: a must be an integer

>>> add_integer(4, "School")
Traceback (most recent call last):
    ...
TypeError: b must be an integer

>>> add_integer(None)
Traceback (most recent call last):
    ...
TypeError: a must be an integer

>>> add_integer(5, None)
Traceback (most recent call last):
    ...
TypeError: b must be an integer

>>> add_integer([1, 2, 3], 4)
Traceback (most recent call last):
    ...
TypeError: a must be an integer

>>> add_integer(1, {"num": 5})
Traceback (most recent call last):
    ...
TypeError: b must be an integer

>>> add_integer((3, 4), 1)
Traceback (most recent call last):
    ...
TypeError: a must be an integer

>>> add_integer(1, (3, 4))
Traceback (most recent call last):
    ...
TypeError: b must be an integer

>>> add_integer(100, -2)
98

>>> add_integer(nan)
Traceback (most recent call last):
    ...
NameError: name 'nan' is not defined

>>> add_integer("Nan",)
Traceback (most recent call last):
    ...
TypeError: a must be an integer

>>> add_integer(-100, -2)
-102

>>> add_integer("hello", "test")
Traceback (most recent call last):
    ...
TypeError: a must be an integer

>>> add_integer("Nan",)
Traceback (most recent call last):
    ...
TypeError: a must be an integer

>>> add_integer(1.1, 2.1)
3

>>> add_integer(True, 2)
3